omero/downloads/cli/help.out

```diff
@@ -47,5 +47,5 @@ Examples:
   $ importer-cli -s localhost -u username -w password -d 50 --debug ALL foo.tiff
 
 For additional information, see:
-http://www.openmicroscopy.org/site/support/omero5.2/users/cli/import.html
+http://docs.openmicroscopy.org/latest/omero/users/cli/import.html
```

Comment:
It should have been omero5 and not omero5.2 so it did get updated
Point at the latest makes more sense than to a specific version